tests/openassetio/hostAPI/test_manager.py

```diff
@@ -262,6 +262,9 @@ def an_entity_spec():
 def some_entity_specs():
     return [EntitySpecification(), EntitySpecification()]
 
+@pytest.fixture
+def some_entity_trait_sets():
+     return [("blob",), ("blob", "image")]
```

Comment:
[fix] I note Github hasn't run checks :angry:  Running pylint locally finds a couple of issues.
```
tests/openassetio/hostAPI/test_manager.py:267:0: W0311: Bad indentation. Found 5 spaces, expected 4 (bad-indentation)
tests/openassetio/hostAPI/test_manager.py:395:0: C0301: Line too long (103/99) (line-too-long)
```

Reply:
Doh! thanks. Fixed in rebase, but GitHub still doesn't seem to want to run any tests...

Reply:
It ran [_some_](https://github.com/TheFoundryVisionmongers/OpenAssetIO/actions/runs/2035699014) tests 🤷 but not the all the ones listed 🙃

Reply:
Actions have finally worked up, `pylint` passes [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/runs/5695151493?check_suite_focus=true).